docs/source/notebooks/climex.ipynb

```diff
@@ -161,7 +161,7 @@
        "}\n",
```

Comment:
<p>From 7 mins to 32 sec !!!  Did you optimize the notebook code, or there are new optimizations in the new xclim or this is the new Pavics server?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>This is one of the biggest benefits of the new pavics server / SSD drives I've seen so far.  Currently working with David to make this notebook a true example of 'Big Data' analysis (time will likely bump back up but will be much more interesting)</p>